docs/schedules.md

```diff
@@ -111,16 +112,16 @@ local and UTC time:
 
 The following aspects of `RFC5545` are _not_ supported by AWX schedules:
 
+* The RRULE must start with the `DTSTART` attribute
 * Strings with more than a single `DTSTART:` component
-* Strings with more than a single `RRULE` component
-* The use of `FREQ=SECONDLY` in an `RRULE`
-* The use of more than a single `FREQ=BYMONTHDAY` component in an `RRULE`
-* The use of more than a single `FREQ=BYMONTHS` component in an `RRULE`
-* The use of `FREQ=BYYEARDAY` in an `RRULE`
-* The use of `FREQ=BYWEEKNO` in an `RRULE`
-* The use of `FREQ=BYWEEKNO` in an `RRULE`
-* The use of `COUNT=` in an `RRULE` with a value over 999
-
+* At least one RRULE must be in the string
```

Comment:
A little confused - isn't this a list of things that are _not_ supported? This looks like a true statement.

Reply:
The wording here has been updated.